spec/tasks/whitehall_news_importer_spec.rb

```diff
@@ -15,12 +15,45 @@
             translations: [
               { locale: "en", title: "Title", summary: "Summary", body: "Body" },
             ],
+            lead_organisations: [
+              { content_id: SecureRandom.uuid },
+              { content_id: SecureRandom.uuid },
+            ],
+            supporting_organisations: [
+              { content_id: SecureRandom.uuid },
+              { content_id: SecureRandom.uuid },
+            ],
+            worldwide_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            topical_events: [SecureRandom.uuid, SecureRandom.uuid],
+            world_locations: [SecureRandom.uuid, SecureRandom.uuid],
           },
         ],
       },
     ].to_json
 
     expect { Tasks::WhitehallNewsImporter.new(JSON.parse(import_json)).import }
       .to change { Document.count }.by(1)
+
+    imported_edition = JSON.parse(import_json)[0]["editions"][0]
+    document_associations = Document.last.associations
+
+    expect(Document.last.summary)
+      .to eq(imported_edition["translations"][0]["summary"])
+    expect(document_associations["primary_publishing_organisation"][0])
+      .to eq(imported_edition["lead_organisations"][0]["content_id"])
```

Comment:
I'm not sure why the org JSON is different to the other things - can we check with @kevindew and look at simplifying the export JSON so it's easier to write the import code?

Reply:
Yeah I think that could be simplified - it was wrote while we still anticipated storing the organisation name.

The ministers one is also complex but for different reasons to store roles/people

Reply:
I've got a PR here to simplify this and just return content_ids for orgs https://github.com/alphagov/whitehall/pull/4316